pointerevents/META.yml

```diff
@@ -191,8 +191,7 @@ links:
         - test: pointerevent_capture_suppressing_mouse.html
           subtest: Test pointer capture.
         - test: pointerevent_capture_suppressing_mouse.html
-    - url: ""
-      label: interop-2023-events
+    - label: interop-2023-events
```

Comment:
Wondering if it should be 'interop-2024-events' instead?  I guess we don't plan to maintain the 2023 dashboard view, right?